azurerm/internal/services/compute/dedicated_host_group_data_source.go

```diff
@@ -61,7 +62,7 @@ func dataSourceArmDedicatedHostGroupRead(d *schema.ResourceData, meta interface{
 	name := d.Get("name").(string)
 	resourceGroupName := d.Get("resource_group_name").(string)
 
-	resp, err := client.Get(ctx, resourceGroupName, name)
+	resp, err := client.Get(ctx, resourceGroupName, name, compute.InstanceView)
```

Comment:
`InstanceView` returns a different response from the API - presumably this wants to be an empty string?

Reply:
Oops, sure it does make sense. Will change it right away